### ISSUE TITLE
Cyberpunk 2077 fails to launch when the iGPU is enabled

### ISSUE THREAD

**Issue by BelleNottelling** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#331]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Cyberpunk 2077

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

i7 13700K

### Graphic Card [Required]

Arc A770 LE 16GB

### GPU Driver Version [Required]

- [X] 31.0.101.4382
- [ ] 31.0.101.4369
- [ ] 31.0.101.4335
- [ ] 31.0.101.4314
- [ ] 31.0.101.4311
- [ ] 31.0.101.4257
- [ ] 31.0.101.4255
- [ ] 31.0.101.4148
- [ ] 31.0.101.4146
- [ ] 31.0.101.4125
- [ ] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/11630475/igcit_ssu.zip)

### Description and steps to reproduce [Required]

1. Have your intel iGPU enabled
2. Try to launch Cyberpunk 2077. It will crash before it reaches the main menu screen
3. Disable iGPU
4. Try to launch Cyberpunk 2077. It will now work as expected

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [ ] High
- [X] Ultra

### Game resolution [Required]

2560x1440

### Game VSync [Required]

On

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

Ultra with motion blur and some other annoying post processing effects disabled. XeSS set to the quality preset.

**Comment by Karen-Intel** [comment]
Hey @BelleNottelling 
I am running the game right now with a 13900K with UHD 770 enabled + A750 with the latest driver and hasn't crashed on me (I have GPU enabled most of the time)[ Here's how it looks for me.](https://www.youtube.com/watch?v=k1bmKn378Wg)
I also made sure to have similar settings to yours (everything in Ultra, Raytracing on, XeSS) and the same resolution, no issues so far so...
A few questions (I recommend you check them one step at a time, run the game and take notes)

0. Can you check if everything is physically well plugged? In the past we have had issues related to crashes because the power cables were not plugged correctly (Heavy games like MS Flight simulator)
1. Check if the game is mapped to High performance on Windows: System > Display > Graphics > Cyberpunk 2077 > Graphics preference > High performance
3. How do you disable de iGPU? Through OS only or BIOS?
4. Talking about the BIOS there's a very recent [update,](https://www.asrock.com/MB/Intel/Z790%20Pro%20RS%20WiFi/index.asp#BIOS) (yesterday) for your board. Please update it to the latest.
5. After the game crashed, did you obtain a dump? If so, please share.
6.  What's the last Windows update you have installed?

I'll be still doing some more testing but if you could share this info, it would help me a lot

Thank you! Talk to you soon

Karen

**Comment by BelleNottelling** [comment]
Hello, @Karen-Intel 😄 
Your little video was more helpful than expected, as you specifically checked the driver version for the iGPU.
I did that and it turns out Windows had automatically installed a driver from December 2022 for the iGPU as when I had installed the drivers for my A770, the iGPU was disabled. Having re-run the driver installer for `4382` and verifying both devices are running that driver version, it appears to be working as expected now.

Thank you for your time!
